tests/component/test_task_properties.py

```diff
@@ -35,3 +36,11 @@ def test___get_devices___shared_interpreter(ai_task: Task):
     devices = ai_task.devices
 
     assert all(dev._interpreter is ai_task._interpreter for dev in devices)
+
+
+def test___add_global_channels___returns_channels(task: Task):
```

Comment:
add_global_channels returns None. It doesn't return the channel object like add_ai_voltage_chan() does.

I would rename this to something like `test___task___add_global_channels___adds_to_channel_names` or `test___task___add_global_channels___channel_names_returns_channels`

Reply:
Changed to `test___task___add_global_channels___adds_to_channel_names`